bom/application/pom.xml

```
@@ -123,7 +123,7 @@
         <postgresql-jdbc.version>42.3.3</postgresql-jdbc.version>
         <mariadb-jdbc.version>2.7.5</mariadb-jdbc.version>
         <mysql-jdbc.version>8.0.28</mysql-jdbc.version>
-        <mssql-jdbc.version>7.2.2.jre8</mssql-jdbc.version>
+        <mssql-jdbc.version>10.2.0.jre8</mssql-jdbc.version>
```

```suggestion
        <mssql-jdbc.version>10.2.0.jre11</mssql-jdbc.version>
```

I think we could use the JRE 11 version since JDK 11 is the minimum to run Quarkus, WDYT @Sanne ?

---

yes! nice catch